dotnetflix.Web/Pages/Tickets/TicketsPage.razor

```diff
@@ -74,7 +82,7 @@
             SelectedShow = await ShowService.GetShow(Id);
             Movie = await MovieService.GetMovie(SelectedShow.MovieId);
             TicketTypes = await TicketService.GetTicketTypes();
-            ChildDiscountAvailable = Movie.SpokenLanguages == "NL" && SelectedShow.Date.TimeOfDay.Hours < 18;
+            //ChildDiscountAvailable = Movie.SpokenLanguages == "NL" && SelectedShow.Date.TimeOfDay.Hours < 18;
```

Comment:
Ik kan dit in de huidige branch niet controleren, maar kijk ik er overheen of wordt ChildDiscountAvailable nu niet geset? En krijg je dus nooit de optie om een ticket met kinder korting te kopen?